### PR TITLE
Brief output

### PR DESCRIPTION
**Pull request by VoxSecundus**
This PR changes the default output of the application to resemble the following codeblock. The old output has been moved, and can be retrieved by using the `verbose` command line argument. 

```
2020-10-21 16:19:38
Nodes

0 node(s) are idle
0 active node(s) have no jobs in any of their partitions

Jobs

2 total job(s) are running (0 have been running for more than 7days
2 total job(s) pending (0 total job(s) not estimated to start within 12hrs after submission)
Estimated time all jobs completed: 2020-10-21 16:58:39 +0100
```
Logically, these report items are identical to the verbose output. However, most of the superfluous data has been omitted.

### CODE REVIEW

File: queue_status.rb [hunk 1]
```diff
@@ -286,43 +286,68 @@ def duplicates(list, part, index)
...
+  ].compact
+
+else
+  msg = ["*#{Time.now.strftime("%F %T")}*\n",
+         "*Nodes*\n\n",
+         "#{idle.length} node(s) are idle",
+         (": #{idle.join(", ")}" if idle.any?),
+         "\n",
+         "#{no_jobs_in_partitions.length} active node(s) have no jobs in any of their partitions",
+         (": #{no_jobs_in_partitions.join(", ")}" if no_jobs_in_partitions.any?),
```
Comment (timalces) [inline queue_status.rb:334]: Perhaps we should also be highlighting any nodes that are down?

File: queue_status.rb [hunk 2]
```diff
@@ -286,43 +286,68 @@ def duplicates(list, part, index)
...
+         "\n\n",
+         "*Jobs*\n\n",
+         "#{total_running} total job(s) are running",
+         (" (#{total_long_running.length} have been running for more than #{formatted_threshold(running_threshold)}"),
```
Comment (timalces) [inline queue_status.rb:338]: Missing a closing bracket in the text here.

File: queue_status.rb [hunk 3]
```diff
@@ -286,43 +286,68 @@ def duplicates(list, part, index)
...
+         "\n",
+         "#{total_pending} total job(s) pending",
+         (" (#{total_long_waiting.length} total job(s) not estimated to start within #{formatted_threshold(wait_threshold)} after submission)"),
```
Comment (timalces) [inline queue_status.rb:341]: To be consistent with the verbose message, this should be "estimated not to start". We're also not highlighting if there any any jobs we can't estimate their start times for (if that's something we want).

**Review by timalces — CHANGES_REQUESTED** [review]

**Comment by VoxSecundus** [comment]
These comments have now been addressed :heavy_check_mark:

File: queue_status.rb [hunk 4]
```diff
@@ -286,43 +286,77 @@ def duplicates(list, part, index)
...
+         "\n\n",
+         "*Jobs*\n\n",
+         "#{total_running} total job(s) are running",
+         (" (#{total_long_running.length} have been running for more than #{formatted_threshold(running_threshold)})"),
+         "\n",
+         "#{total_pending} total job(s) pending",
+         (" (#{total_long_waiting.length} total job(s) estimated not to start within #{formatted_threshold(wait_threshold)} after submission)"),
+         ("Insufficient data to estimate job start times" if no_start_data),
+         "\n"
+         ("#{total_long_waiting.length} total job(s) estimated not to start within #{formatted_threshold(wait_threshold)} after submission" if !no_start_data),
```
Comment (timalces) [inline queue_status.rb:347]: The number of long waiting jobs are being displayed twice if this condition is met.

File: queue_status.rb [hunk 5]
```diff
@@ -286,43 +286,77 @@ def duplicates(list, part, index)
...
+         (": #{no_jobs_in_partitions.join(", ")}" if no_jobs_in_partitions.any?),
+         "\n\n",
+         "*Jobs*\n\n",
+         "#{total_running} total job(s) are running",
+         (" (#{total_long_running.length} have been running for more than #{formatted_threshold(running_threshold)})"),
+         "\n",
+         "#{total_pending} total job(s) pending",
+         (" (#{total_long_waiting.length} total job(s) estimated not to start within #{formatted_threshold(wait_threshold)} after submission)"),
+         ("Insufficient data to estimate job start times" if no_start_data),
+         "\n"
```
Comment (timalces) [inline queue_status.rb:346]: Missing a comma at the end of this line.

File: queue_status.rb [hunk 6]
```diff
@@ -286,43 +286,77 @@ def duplicates(list, part, index)
...
+         "#{no_jobs_in_partitions.length} active node(s) have no jobs in any of their partitions",
+         (": #{no_jobs_in_partitions.join(", ")}" if no_jobs_in_partitions.any?),
+         "\n\n",
+         "*Jobs*\n\n",
+         "#{total_running} total job(s) are running",
+         (" (#{total_long_running.length} have been running for more than #{formatted_threshold(running_threshold)})"),
+         "\n",
+         "#{total_pending} total job(s) pending",
+         (" (#{total_long_waiting.length} total job(s) estimated not to start within #{formatted_threshold(wait_threshold)} after submission)"),
+         ("Insufficient data to estimate job start times" if no_start_data),
```
Comment (timalces) [inline queue_status.rb:345]: This needs an` \n` at the front.

File: queue_status.rb [hunk 7]
```diff
@@ -286,43 +286,77 @@ def duplicates(list, part, index)
...
+         "\n",
+         "#{down.length} node(s) are down",
+         (": #{down.join(", ")}" if down.any?),
+         "\n",
+         "#{no_jobs_in_partitions.length} active node(s) have no jobs in any of their partitions",
+         (": #{no_jobs_in_partitions.join(", ")}" if no_jobs_in_partitions.any?),
+         "\n\n",
+         "*Jobs*\n\n",
+         "#{total_running} total job(s) are running",
+         (" (#{total_long_running.length} have been running for more than #{formatted_threshold(running_threshold)})"),
```
Comment (timalces) [inline queue_status.rb:341]: Do we want to have the option to show the ids of these as well (if the ids arg given)?

**Review by timalces — CHANGES_REQUESTED** [review]
Sorry, also just realised that the Readme needs updating to reflect these changes.

**Comment by VoxSecundus** [comment]
Further updates :heavy_check_mark:

File: queue_status.rb [hunk 8]
```diff
@@ -286,43 +286,78 @@ def duplicates(list, part, index)
...
+         "*Jobs*\n\n",
+         "#{total_running} total job(s) are running",
+         (" (#{total_long_running.length} have been running for more than #{formatted_threshold(running_threshold)}" if total_long_running.any?),
+         (": #{total_long_running.map {|job| job[1] }.join(", ") }" if total_long_running.any? && show_ids),
+         (")" if total_long_running.any?),
+         "\n",
+         "#{total_pending} total job(s) pending",
+         (" (Insufficient data to estimate job start times)" if no_start_data),
+         (" (#{total_long_waiting.length} total job(s) estimated not to start within #{formatted_threshold(wait_threshold)} after submission)" if !no_start_data),
+         "\n",
```
Comment (timalces) [inline queue_status.rb:348]: Sorry, last point! Think this `\n`should be removed now.

**Pull request merged**